src/serverCapabilities.ts

```diff
@@ -0,0 +1,136 @@
+/*
+Copyright 2024 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { IHttpOpts, MatrixHttpApi, Method } from "./http-api";
+import { logger } from "./logger";
+
+// How often we update the server capabilities.
+// 6 hours - an arbitrary value, but they should change very infrequently.
```

Comment:
I thought it was 8 hours.
Android is doing 8 saying to match web
https://github.com/element-hq/element-android/blob/097d3923aa03b44c02888cc94a197d34e5e5f780/matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/homeserver/GetHomeServerCapabilitiesTask.kt#L192

I supposed this is not spec'ed anywhere?

Reply:
Whoever wrote the js-sdk part picked 6 I guess. It's not specced and it doesn't really matter, I don't think there are any requirements on clients caching it at all.